.travis.yml

```diff
@@ -14,10 +14,11 @@ addons:
       -
         sourceline: "ppa:marutter/rrutter"
 script:
-  - "bazel build //main:feed_data" 
+  - "bazel build //main:feed_data"
   - "bazel test //tests:main_test"
+  - "bazel test //tests:test_GH_model"
 sudo: false
 
 after_failure:
   - cat /home/travis/.cache/bazel/_bazel_travis/b780598bdbf255f7df0b39f918df3247/execroot/__main__/bazel-out/k8-fastbuild/testlogs/tests/main_test/test.log
-
+  - cat /home/travis/.cache/bazel/_bazel_travis/b780598bdbf255f7df0b39f918df3247/execroot/__main__/bazel-out/k8-fastbuild/testlogs/tests/test_apply_GH_model/test.log
```

Comment:
I wonder whether b780598bdbf255f7df0b39f918df3247 is going to be robust across test runs

Reply:
I guess we'll see

Reply:
Oh that's a good point....